src/components/header/header.js

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Icon } from 'react-fa';
+import './header.scss';
+import Logo from '../../components/logo/logo';
+import SlackLogo from '../../images/Slack-Monochrome-White.svg';
+
+export default class Hello extends React.Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    return (
+      <div className="row-fluid">
+        <div id="header" className="col-xs-12">
+          <div className="col-xs-2 logo">
+            <Logo />
+          </div>
+
+          <div className="col-xs-7">&nbsp;</div>
```

Comment:
This seems kinda hacky. Instead, you can give `.col-xs-3` another `offset` class. I think it'd be `.col-xs-offset-7` and drop this empty div. Also, bootstrap 3 still has `pull-right` available.